docuilib/src/components/ComponentPage.tsx

```diff
@@ -79,6 +79,7 @@ export default function ComponentPage({component}) {
         // title: name,
         layout: 'horizontal',
         ...hero,
+        title: component.name,
```

Comment:
Look at the commented out code. We don't to add the 'incubator' prefix to the incubator components?

Reply:
I don't see why not
We want developers to know how to access Incubator components, especially if we still have incubator and non-incubator version (like Dialog and Slider)

Reply:
So use the commented 'name' in the title filed, no?